homework-submission/react-week2/src/components/TodoItems/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+export default function TodoItems({name, done, buttonText, handleCheckboxChange, handleRemove}) {
+    const isDone = () =>{
+        return{
+            textDecoration: done ? 'line-through' : ''
+        }
+    }
+
+
+    return (
+        <li style={isDone()}>
```

Comment:
Better to give a className based on if the item is done or not. This way you have a function which seems to be general (`isDone`), but actually returns a very specific styling value.